app/uk/gov/hmrc/incometaxstatebenefitsfrontend/config/AppConfig.scala

```diff
@@ -1,10 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package uk.gov.hmrc.incometaxstatebenefitsfrontend.config
 
-import javax.inject.{Inject, Singleton}
 import play.api.Configuration
 
+import javax.inject.{Inject, Singleton}
+
 @Singleton
 class AppConfig @Inject()(config: Configuration) {
-  val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
 
+  val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
```

Comment:
If you're adding this to `AppConfig`, then ideally the entry needs to be added to `conf/application.conf`. But it's not required as you've got the `getOrElse` call there.

Reply:
This is a bare bones app and the code is generated (not added by me). This will change. I only need to push those now so that I will have a green build. The default (automatically) created build fails

Reply:
Right. It's defined in the file already. Marking the conversation as resolved.